lib/linguist/vendor.yml

```diff
@@ -302,3 +302,6 @@
 
 # Android Google APIs
 - (^|/)\.google_apis/
+
+# Jenkins Pipeline
+- ^Jenkinsfile$
```

Comment:
Typically a `Jenkinsfile` will be at the root of its repository; when used as part of a multibranch Pipeline project, that is required as this exact filename is the indication that a repository/branch is ready to build.